tests/build-std/main.rs

```diff
@@ -114,6 +114,7 @@ fn basic() {
         // Importantly, this should not say [UPDATING]
         // There have been multiple bugs where every build triggers and update.
         .with_stderr_data(str![[r#"
+[UPDATING] crates.io index
```

Comment:
Looked a few lines above

```
        // Importantly, this should not say [UPDATING]
        // There have been multiple bugs where every build triggers and update.
```

Before proceeding, we may want to figure out what has changed and if it is safe to update index.

Reply:
good call

Reply:
Perhaps rust-lang/rust#128534, though I may not have time looking into it today.